pkgs/tools/audio/bark/default.nix

```
@@ -0,0 +1,29 @@
+{ rustPlatform
+, fetchFromGitHub
+, lib
+, pkg-config
+, alsa-lib
+, speexdsp
+}:
+rustPlatform.buildRustPackage {
```

```suggestion
{ lib
, rustPlatform
, fetchFromGitHub
, pkg-config
, alsa-lib
, speexdsp
}:

rustPlatform.buildRustPackage {
```

According to coding convention, lib should come first.

> Arguments should be listed in the order they are used, with the exception of lib, which always goes first.

See https://ryantm.github.io/nixpkgs/contributing/coding-conventions/

---

Heh. 

So am I changing this or not? :-)

It passes `nixpkgs-fmt` ;)

---

Feel free to dismiss this if you want :)

But you will still have to fix the `vendorHash` and update your commit message from
`bark: init at 2023-08-24` to
`bark: init at unstable-2023-08-22`

---

Both done!

---

I don't understand how the cargo hash changed thought...